### PR TITLE
Curve Fever Pro

### PR DESCRIPTION
**Pull request by Blyster111**
Discord Presence For Game 'Curve Fever Pro'
cfpro discord

### CODE REVIEW

File: websites/C/Curve Fever Pro/.gitattributes [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Auto detect text files and perform LF normalization
+* text=auto
```
Comment (ririxichan) [inline websites/C/Curve Fever Pro/.gitattributes:2]: delete this file

File: websites/C/Curve Fever Pro/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "name": "Robin",
+    "id": "284370397535666188"
+  },
+  "url": "curvefever.pro",
+  "description": {
+    "english": "The smash hit Curve Fever is back with a new multiplayer browser game: Curve Fever Pro. Cut off opponents, squeeze through tiny holes and unlock new powers."
+  },
+  "service": "Curve Fever Pro",
+  "logo": "https://robinko.eu/premid%20assets/cfprologo.png",
+  "thumbnail": "https://robinko.eu/premid%20assets/cfprothumbnail.png",
```
Comment (ririxichan) [inline websites/C/Curve Fever Pro/dist/metadata.json:13]: upload to imgur and use i.imgur links

File: websites/C/Curve Fever Pro/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "Robin",
+    "id": "284370397535666188"
+  },
+  "url": "curvefever.pro",
+  "description": {
+    "english": "The smash hit Curve Fever is back with a new multiplayer browser game: Curve Fever Pro. Cut off opponents, squeeze through tiny holes and unlock new powers."
```
Comment (ririxichan) [inline websites/C/Curve Fever Pro/dist/metadata.json:9]: english -> en

**Review by ririxichan — CHANGES_REQUESTED** [review]
fix codacy errors too :)

**Review by Bas950 — APPROVED** [review]

**Review by ririxichan — APPROVED** [review]

**Pull request merged**